Solutions/DigitalGuardianDLP/Hunting Queries/DigitalGuardianRareNetworkProtocols.yaml

```diff
@@ -0,0 +1,29 @@
+id: 8ab2f0db-baa1-495c-a8dd-718b81d0b8c7
+name: Digital Guardian - Rare network protocols
+description: |
+  'Query searches rare network protocols.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: DigitalGuardianDLP
+    dataTypes:
+      - DigitalGuardianDLPEvent
+tactics:
+  - Exfiltration
+relevantTechniques:
+  - T1048
+query: |
+  DigitalGuardianDLPEvent
+  | where TimeGenerated > ago(24h)
+  | where isnotempty(NetworkApplicationProtocol)
+  | summarize count() by SrcIpAddr, SrcUserName
+  | order by count_ asc

```

Comment:
How are we determining which protocols are rare. Wondering if we wanted to use
 top 5 or top 10 here.

Reply:
fixed